src/api/environment.cc

```diff
@@ -660,23 +665,62 @@ Maybe<bool> InitializePrimordials(Local<Context> context) {
   Local<String> global_string = FIXED_ONE_BYTE_STRING(isolate, "global");
   Local<String> exports_string = FIXED_ONE_BYTE_STRING(isolate, "exports");
 
+  Local<String> messaging_clone_symbol_string =
```

Comment:
It would be nicer if these are moved to a method in `worker::JSTransferable` which returns an object with the symbols and the JSTransferable  constructor

Reply:
@joyeecheung thanks, done in https://github.com/nodejs/node/pull/41148/commits/9353cafcdf9d5d8314e50af2d505473c14cd04f4